test/withdraw.test.ts

```diff
@@ -347,74 +341,6 @@ describe('StakingPool.withdraw', () => {
         expect(poolDataAfterWithdraw).eql(expectedPoolDataWithdraw);
         expect(userDataAfterWithdraw).eql(expectedUserDataWithdraw);
       });
-
-
-      context('when admin sets bob as a manager', async () => {
-        it('bob becomes a manager and extend the pool and alice withdraw', async () => {
```

Comment:
I thought `setManager` does not have to be tested in `stake.test.ts`, `withdraw.test.ts`, `claim.test.ts`, so extracted the tests in `stake.test.ts` into `extendPoolTest.ts`. 
If you think it is not enough, please comment.

Reply:
I totally agree with you.